cfme/automate/buttons.py

```diff
@@ -331,7 +331,8 @@ def create(self):
         })
         view.add_button.click()
         view = self.create_view(ButtonGroupDetailView, self.group)
-        assert view.is_displayed
+        # TODO: Enable this
```

Comment:
@mshriver I want ot make a followup PR that will introduce Collections and while doing that it will be the best way to streamline the view detection here.